config/router.js

```diff
@@ -21,13 +21,41 @@ router.use((req, res, next) => {
 });
 
 /**
- * Chatbot routes.
+ * Chatbot.
  */
 const chatbotRouter = rootRequire('config/router-chatbot');
 router.use('/v1/chatbot', chatbotRouter);
 
 /**
- * Legacy routes.
+ * Campaigns.
+ */
+router.get('/v1/campaigns', (req, res) => {
+  const findClause = {};
+  if (req.query.campaignbot) {
+    const campaignConfigVar = process.env.CAMPAIGNBOT_CAMPAIGNS;
+    const campaignIDs = campaignConfigVar.split(',').map(id => Number(id));
+    findClause._id = { $in: campaignIDs };
+  }
+
+  return app.locals.db.campaigns
+    .find(findClause)
```

Comment:
Is there documentation for this API route to indicate all campaigns are returned if you don't pass campaign bot?